Drivers/seds/SunSensorComponent.xml

```diff
@@ -0,0 +1,98 @@
+        <LongDescription>SEDS SunSensor driver implementation</LongDescription>
+        <ProvidedInterfaceSet>
+          <Interface name="SunSensor" type="com/n7space/SunSensor/SunSensorInterfaceType"></Interface>
+        </ProvidedInterfaceSet>
+        <RequiredInterfaceSet>
+          <Interface name="AfecHwas" type="com/n7space/AfecHwas/AfecHwasInterfaceType" />
+          <Interface name="SunSensorReturn" type="com/n7space/SunSensor/SunSensorReturnInterfaceType" />
+        </RequiredInterfaceSet>
+        <Implementation>
+          <VariableSet>
+            <Variable name="afecVar" type="com/n7space/AfecHwas/AfecHwas"></Variable>
+            <Variable name="afecConfigVar" type="com/n7space/AfecHwas/AfecHwasConfig" />
+            <Variable name="chOutputVar" type="com/n7space/AfecHwas/ConversionData"></Variable>
+          </VariableSet>
+          <ActivitySet>
+            <Activity name="InitSunSensor">
+              <Body>
+                <Assignment outputVariableRef="afecConfigVar.mAfecInstance">
+                  <LongDescription>Set Afec instance to instance 0</LongDescription>
+                  <Value value="AfecHwas_Instance_Afec0" />
+                </Assignment>
+                <Assignment outputVariableRef="afecConfigVar.mStartupTime">
+                  <Value value="3" />
+                </Assignment>
+                <Assignment outputVariableRef="afecConfigVar.mPrescalerValue">
+                  <Value value="2" />
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Configure Afec" interface="AfecHwas" command="InitInstance">
+                  <ArgumentValue name="afec">
+                    <VariableRef variableRef="afecVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="config">
+                    <VariableRef variableRef="afecConfigVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+              </Body>
+            </Activity>
+            <Activity name="RequestData">
+              <Body>
+                <SendCommandPrimitive command="GetValueCmd" interface="AfecHwas" shortDescription="Sun sensor is connected to PA19. It is AFE0_AD8. On Xplained Ultra board it is port J502 pin 3">
+                  <ArgumentValue name="afec">
+                    <VariableRef variableRef="afecVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="analogChannel">
+                    <Value value="8" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+              </Body>
+            </Activity>
+            <Activity name="ReadData">
+              <Argument name="chOutput" type="com/n7space/AfecHwas/ConversionData"></Argument>
+              <Body>
+                <SendCommandPrimitive command="ReturnDataCmd" interface="SunSensorReturn" shortDescription="Sun sensor is connected to PA19. It is AFE0_AD8. On Xplained Ultra board it is port J502 pin 3">

```

Comment:
Can we do at least some "demo" processing... like normalization to 0-100 range or something. It does not need to be calibrated (although, use of Calibration would be cool).

Reply:
Sure, I can do that.